samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/RabbitMQTransportAdapter.sln.DotSettings

```diff
@@ -1,6 +0,0 @@
-<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Yes we use r# settings in samples since it allows us to keep the code consistent